src/RageSoundMixBuffer.h

```diff
@@ -26,8 +26,9 @@ class RageSoundMixBuffer
 
 private:
 	float *m_pMixbuf;
-	unsigned m_iBufSize; /* actual allocated samples */
-	unsigned m_iBufUsed; /* used samples */
+	float *m_pMixbufBackup;
```

Comment:
`m_pMixbufBackup` is only used in `Extend()` function so it can be a local variable.

Reply:
True, I thought it made sense to define it here though.